src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_SelectAll.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Editor.Commands;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion
+{
+    internal partial class Controller
+    {
+        CommandState ICommandHandler<SelectAllCommandArgs>.GetCommandState(SelectAllCommandArgs args, System.Func<CommandState> nextHandler)
+        {
+            AssertIsForeground();
+            return nextHandler();
+        }
+
+        void ICommandHandler<SelectAllCommandArgs>.ExecuteCommand(SelectAllCommandArgs args, Action nextHandler)
+        {
+            AssertIsForeground();
+            if (sessionOpt != null)
+            {
+                StopModelComputation();
```

Comment:
Guess what "StopModelComputation" calls and how it decides if it should do anything.

But yeah, this should probably be consistent.